doc/user/content/sql/functions/windowed-aggregates.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Windowed Aggregate Functions"
+description: "Aggregates timestamp data into well-defined windows."
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+When handling [`timestamp`]-valued data, you might want to perform aggregate
+computations on values that occur within the same window of time, e.g. in
+non-overlapping 5-minute intervals.
+
+To support this, Materialize offers what we call "windowed aggregate functions."
+These functions calculate which windows of time each row belongs to, providing a
+value to `GROUP BY` in in aggregate functions.
+
+Note that these functions greatly differ from Materialize's [windowing
+idioms](/guides/temporal-filters/#windowing-idioms). Our windowing idioms let
+users control which data views return as a function of `mz_logical_timestamp()`,
+whereas windowed aggregate functions' primary function is to express `timestamp`
+data as well-defined windows.
+
+{{< experimental v0.11.1 />}}
```

Comment:
Should this be v0.11.1 or v0.12.0? We only increment the third number for urgent bug fix releases now.

Reply:
Oh, I just grabbed the most recent version number. Will fix.